boltdesignsystem/bolt
# Add "locked" badge

## Jira

http://vjira2:8080/browse/BDS-2114

## Summary

Add "locked" Badge template to Blueprints.

## Details

Add "locked" option to Badge template that fades out the badge and displays a lock over it.

Added a new utility class, `u-bolt-absolute-center`, to help set this up. Also removed some unnecessary grid and utility classes from the Badge template.

## How to test

- Review files changed
- Do you approve of the [new utility class](https://github.com/boltdesignsystem/bolt/pull/1847/files#diff-1c64ddc1abae139b0b22adc69eb8fbb2)?
- See "locked" badge demo on feature branch: `/pattern-lab/patterns/03-blueprints-05-pages-t1-landing-pages-mission-landing-t1-mission-landing/03-blueprints-05-pages-t1-landing-pages-mission-landing-t1-mission-landing.html` (awaiting deploy)
